packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -112,10 +122,14 @@ const Cell = (props) => {
     }
   };
 
-  const handleMouseDown = () => {
+  const handleMouseDown = (event) => {
     setFocusedCell({ rowId, columnId: id });
     if (isSelectableCell && onCellSelect) {
-      onCellSelect({ rowId, columnId: id });
+      event.preventDefault();
+      if (onCellSelect) {
+        onCellSelect({ rowId, columnId: id });
```

Comment:
If a cell is not selectable, we should not trigger the callback, correct?

Reply:
That is correct, there is if statement to check if a cell is selectable above:https://github.com/cerner/terra-framework/blob/75f1f241355ea225b2b4cfb4ef22ac221df279d3/packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx#L127-L130